### DISCUSSION TITLE
Do not require org's name in 'repo' if a scope is set to an org

### DISCUSSION THREAD

**Discussion by martynasp-wix** [community/community#10040] [Other Feature Feedback, Questions, & Ideas] [answered]
If I set the scope to an organisation `my-org` and then try to search in a repo `my-repo`, I still have to enter the repo's name in the `repo` parameter, e.g. `repo:my-org/my-repo`, because otherwise I get an error "search failed". If the scope is already set to an organisation, then it should be enough to just say `repo:my-repo` and not type the organisation's name again.

**Comment by colinwm** [comment]
This is a great idea! I'm working on it right now, will update when it's shipped

**Comment by colinwm** [comment, marked as answer]
Done! Thanks again for the suggestion. 

E.g. try something like this: https://cs.github.com/?scopeName=All+repos&scope=&q=org%3Agithub+repo%3Alinguist

**Reply by martynasp-wix** [reply]
Thank you!
